Mathlib/RingTheory/MvPolynomial/WeightedHomogeneous.lean

```diff
@@ -272,7 +272,7 @@ theorem prod {ι : Type*} (s : Finset ι) (φ : ι → MvPolynomial σ R) (n : 
     intro j hjs
     exact h j (Finset.mem_insert_of_mem hjs)
 
-/-- A non zero weighted homogeneous polynomial of weighted degree `n` has weighted total degree
+/-- A nonzero-weighted homogeneous polynomial of weighted degree `n` has weighted total degree
```

Comment:
Could either be "A nonzero weighted" or "A nonzero-weighted". Quite unsure about this one.

Reply:
Hmm, looking at the code it seems "weighted" binds to homogeneous, so I've now changed it to "A nonzero weighted-homogeneous ...". I think that should be right.